auth_signup_verify_email/tests/test_verify_email.py

```diff
@@ -54,19 +42,22 @@ def search_text(self, doc, text):
 
     def test_bad_email(self):
         """Test rejection of bad emails."""
-        self.data["login"] = "bad email"
-        doc = self.html_doc(data=self.data)
+        data = self.data.copy()
```

Comment:
You don't need to change this. `self.setUp()` is ran before each test, and it redeclares `self.data`.

Reply:
I know, it was supposed to go away during rebase/cleanup. Guess I'm doing too many things at the same time :stuck_out_tongue_closed_eyes: